integration_test/build-adapters.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+base_name="gcr.io/${1}/healthcare-api-dicom-dicomweb-adapter-"
+
+build_adapter () {
+  adapter_name=${base_name}${1}
+  version=$(cat build.gradle | grep "version = '"| grep -Eo "[0-9]{1,3}\.[0-9]{1,3}")
+  postfix=$(gcloud container images list-tags "${adapter_name}" --sort-by=~tags | grep " $version" |  grep -Eo "([0-9]{1,3}\.){2}[0-9]{1,3}" | head -1 | grep -Eo "[0-9]{1,3}" | tail -1)
```

Comment:
What if we bump the version from 0.1 to 0.2? Then if the most recent container is 0.1.8, we will write 0.2.9 rather than 0.2.0, right?

Reply:
this expression filters string by $version(0.2 in your example) so postfix will be empty at line 7 and 1 at line 8, so result will be 0.2.1